bash_completion

```diff
@@ -833,7 +833,8 @@ _comp_delimited()
         COMPREPLY=($(compgen "$@" -- "${cur##*"$delimiter"}"))
     fi
 
-    ((${#COMPREPLY[@]} == 1)) && COMPREPLY=(${COMPREPLY/#/$prefix})
+    COMPREPLY=("${COMPREPLY[@]/#/"$prefix"}")
```

Comment:
```suggestion
    ((${#COMPREPLY[@]})) && COMPREPLY=("${COMPREPLY[@]/#/"$prefix"}")
```

Can you also add a test case where no completions are generated in `test_unit_delimited.py`?